### PR TITLE
enhance readme; add params to create_shinylive_url; other minor docs …

### PR DESCRIPTION
**Pull request by pawelru**
- enhance readme:
  - add usage in vignette
  - add gifs
- add `mode` and `header` params to `create_shinylive_url()`
- other minor docs updates

### CODE REVIEW

**Comment by pawelru** [comment]
I have read the CLA Document and I hereby sign the CLA

File: R/parse_url.R [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
+#' @param mode (`character(1)`) A string with mode. One of "app" or "editor".
+#' @param header (`logical(1)`) A logical value indicating whether to include header.
+#' Ignored if `mode` is "editor".
 #' @return (`character(1)`) Shinylive app url.
 #'
 #' @export
 #'
 #' @examples
 #' code <- "this is your app code as a string"
 #' create_shinylive_url(code)
```
Comment (donyunardi) [inline R/parse_url.R:16]:
```suggestion
#' create_shinylive_url(code)
#' create_shinylive_url(code, mode = "editor")
#' create_shinylive_url(code, header = FALSE)
```

Should we consider adding more examples using the new parameters?

File: R/parse_url.R [hunk 2]
```diff
@@ -1,18 +1,23 @@
-#' Creates Shinylive url for the app code.
+#' Creates Shinylive url based on app code.
 #'
 #' @importFrom jsonlite unbox toJSON
 #' @importFrom lzstring compressToEncodedURIComponent
 #'
 #' @param code (`character(1)`) A string with app code.
+#' @param mode (`character(1)`) A string with mode. One of "app" or "editor".
```
Comment (donyunardi) [inline R/parse_url.R:7]: When not defined, the default value for `mode` argument would be `app`. Should this be mentioned here?

**Review by donyunardi — CHANGES_REQUESTED** [review]
The new arguments to the `create_shinylive_url` works fine.
I only have minor comments, but I also think it might be a good idea to add unit tests for the new arguments.

File: R/parse_url.R (hunk 1 above)
Comment (pawelru) [reply R/parse_url.R:16]: yes - added!

File: R/parse_url.R (hunk 2 above)
Comment (pawelru) [reply R/parse_url.R:7]: Added!

**Comment by donyunardi** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by donyunardi — APPROVED** [review]

**Pull request merged**